packs/equipment-effects/effect-ghostbane-fulu.json

```diff
@@ -0,0 +1,64 @@
+{
+    "_id": "02LNcIl70TTy9XS1",
+    "img": "systems/pf2e/icons/equipment/consumables/fulus/fulu-modest.webp",
+    "name": "Effect: Ghostbane Fulu",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Ghostbane Fulu]</p>\n<p>Upon activation, the weapon gains the benefit of the <em>ghost touch</em> property rune against the triggering incorporeal creature for 1 minute.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 6
+        },
+        "rules": [
+            {
+                "key": "TokenMark",
```

Comment:
Do we want to provide totm toggles when using TokenMark?

Reply:
We may want to consider it, but we'd want to maybe have a way to shut off the TokenMark RE itself.